docs/csharp/language-reference/builtin-types/default-values.md

```diff
@@ -1,25 +1,24 @@
 ---
-title: "Default values table - C# reference"
-description: Learn what are the default values of C# types.
```

Comment:
Why the description is deleted ?

Reply:
It hasn't added anything to the title

Reply:
I'd prefer adding back the description, and adding some context that helps someone who sees this page in search results. Something like: 

```yml
- description: Learn the default values of C# types such as bool, char, int, float, double and more.
```

Reply:
@BillWagner thank you for the suggestion; I've updated the metadata